lib/all_your_base_cases.rb

```diff
@@ -57,16 +57,17 @@ def error_message
 
 class AllYourBaseCase::PreProcessor
   class << self
+    attr_reader :row
+
     def call(row)
       @row = row
 
       row.merge('expected' => expected_value)
     end
 
+    private :row
     private
 
```

Comment:
Copying commit message here so it can be read inline:

This change is to remove a warning present in ruby versions
less than 2.3.0. A thread discussing this warning can be found
at https://bugs.ruby-lang.org/issues/10967. It is removed in
ruby/ruby@32a5a09.